src/style/app.module.css

```diff
@@ -338,8 +351,8 @@
     :disabled,
     :checked
   ) {
-  background-color: transparent !important;
-  border: 1px solid var(--brown-color);
+  box-shadow: 2.5px 2.5px 2.5px rgba(0, 0, 0, 0.3);
+  border: 1px solid var(--brown-color) !important;
   color: var(--brown-color) !important;
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with future attempts to having Dark Mode.
- For example `--search-button-bg` is much better than `--light-blue`